test/api/base.js

```diff
@@ -0,0 +1,13 @@
+const assert = require("assert");
+const invoiceExpress = require("../..");
+
+describe("API", function() {
+    it("should be able to instantiate the API", async () => {
+        const api = new invoiceExpress.API({
+            accountName: "name",
+            apiKey: "key"
+        });
+        assert.strictEqual(Boolean(api.accountName), true);
+        assert.strictEqual(Boolean(api.apiKey), true);

```

Comment:
Maybe add one for `assert.strictEqual(Boolean(api.invoiceBaseUrl), true);`
Also, maybe test for `api.accountName === "name"` and same for key